commit Diplom2

ChervenIrina opened the pull request.

Commit:
commit Diplom2

AlexKitIt commented on data/ingredient.py:
```diff
@@ -0,0 +1,10 @@
+
+
+@allure.step('Получаем ингредиенты достпные для заказа')
+def get_ingredients():
```
Методы лучше описывать в классах, что импортировать не модули а классы. Когда в модуле этих методов будет слишком много это станет проблемой

AlexKitIt commented on data/user.py:
```diff
@@ -0,0 +1,48 @@
+
+    return payload
+
+@allure.step('Генерация случай строки (ТД)')
+def generate_random_string(length):
+    letters = string.ascii_lowercase
+    random_string = ''.join(random.choice(letters) for i in range(length))
+    return random_string
```
Нужно исправить: это максимально общий метод, его необходимо расположить в helpers тем самым открывая возможность его использования в других модулях

ChervenIrina replied:
исправлено

Commit:
commit fix:
1. user.py (убрала метод генерации строки)
2. helpers.py (добавила метод генерации строки)

3. обновила requarements
